wixcode-seo/wix-animations/TimeLine.service.json

```diff
@@ -0,0 +1,814 @@
+        "locations":
+          [ { "lineno": 221,
+              "filename": "timeline.js" } ],
+        "docs":
+          { "summary": "Replays a timeline.",
+            "description":
+              [ "Replaying restarts a playing a paused or playing timeline from the ",
+                " beginning of the timeline and sets the [`hidden`]($w.HiddenMixin.html#hidden)",
+                " property of the animated elements to `false`. ",
+                "",
+                " Note that when a timeline is created it is paused by default at the ",
+                " beginning of the timeline." ],
+            "links": [],
+            "examples":
+              [ { "title": "Pause a timeline",
+                  "body":
+                    [ "import wixAnimations from 'wix-animations';",
+                      "",
+                      "let timeline = wixAnimations.timeline();",
+                      "",
+                      "// ...",
+                      "",
+                      "timeline.pause();" ],
+                  "extra":
+                    {  } },
+                { "title": "Create buttons for controlling timeline playback",
+                  "body":
+                    [ "import wixAnimations from 'wix-animations';",
+                      "",
+                      "let timeline = wixAnimations.timeline(",
+                      "  {",
+                      "    \"repeat\": 3,",
+                      "    \"repeatDelay\": 100,",
+                      "    \"yoyo\": true",
+                      "  }",
+                      ");",
+                      "",
+                      "$w.onReady( function () {",
+                      "  const myImage = $w(\"#myImage\");",
+                      "",
+                      "  timeline.add( myImage, {",
+                      "    \"rotate\": 360,",
+                      "    \"scale\": .5,",
+                      "    \"duration\": 1000",
+                      "  } );",
+                      "",
+                      "  timeline.add( myImage, {",
+                      "    \"opacity\": 0,",
+                      "    \"duration\": 1000",
+                      "  } );",
+                      "",
+                      "  $w(\"#playButton\").onClick( () => {",
+                      "    timeline.play();",
+                      "  } );",
+                      "",
+                      "  $w(\"#pauseButton\").onClick( () => {",
+                      "    timeline.pause();",
+                      "  } );",
+                      "",
+                      "  $w(\"#replayButton\").onClick( () => {",
+                      "    timeline.replay();",
+                      "  } );",
+                      "} );" ],
+                  "extra":
+                    {  } } ],
+            "extra":
+              {  } },
+        "extra":
+          {  } } ],
+  "callbacks": [],
+  "messages":
+    [ { "name": "Animation",
```

Comment:
Maybe add another animation example that uses some of the other properties so users can just copy and paste?